README.md

```diff
@@ -144,15 +174,21 @@ Some important modules:
 * [Use cases](usecase) - Business logic 
 * [Adapters](adapter) - Convertors (e.g., parsers and exporters)
 * [application/cli-app](application/cli-app) - The main command-line application
+* [application/web-app](application/web-app) - The web app implementation of the application that relies on SpringBoot framework
```

Comment:
Needs updating. We have web-app and underneath it `spring-server` and `react-client`